.secrets.baseline

```diff
@@ -5122,7 +5140,7 @@
       }
     ]
   },
-  "version": "0.13.1+ibm.61.dss",
```

Comment:
Can you use latest detect secret version

Reply:
Ugh, every single time! Done.